data/commands/index.js

```diff
@@ -0,0 +1,14 @@
+const { readdir } = require('fs').promises
+const { join } = require('path')
+
+async function requireCommands () {
+  const commands = {}
+  const files = await readdir(__dirname)
+  for (let i = 0; i < files.length; i++) {
```

Comment:
Make `i` a constant here since it's not changed mid-iteration.

Reply:
what?

Reply:
No you cannot.
<img width="628" alt="screen shot 2019-01-02 at 11 52 36 am" src="https://user-images.githubusercontent.com/8886322/50602300-ff1be180-0e84-11e9-8ed4-ac6eca82c325.png">

Reply:
Oh, wait, that only applies to the `in` and `of` operators.